packages/orbit-design-tokens/src/dictionary/definitions/component/modal/modal-border.json

```diff
@@ -1,9 +1,26 @@
 {
   "component": {
+    "modal": {
+      "border-color": {
+        "type": "color",
+        "value": "{foundation.palette.cloud.normal}"
+      },
+      "border-radius-mobile": {
+        "type": "border-radius",
+        "value": 12
+      },
+      "border-radius": {
+        "type": "border-radius",
+        "value": 9
+      }
+    },
     "border-color": {
       "modal": {
         "type": "color",
-        "value": "{foundation.palette.cloud.normal}"
+        "value": "{foundation.palette.cloud.normal}",
+        "deprecated": true,
+        "deprecated-replace": "{global.elevation.flat.background}",
```

Comment:
should be `global.elevation.flat.border-color`

Reply:
Yes, my typo